src/applications/ivc-champva/10-7959C/containers/ConfirmationPage.jsx

```diff
@@ -54,9 +69,12 @@ export function ConfirmationPage(props) {
     showMail: true,
     allPages: form.pages,
     fileNameMap: prefixFileNames(data, requiredFiles),
+    optionalDescription: '',
+    requiredDescription: '',
     requiredFiles,
     nonListNameKey: 'applicantName',
     mailingAddress: officeAddress,
+    mailPreamble,
     officeName: office,
```

Comment:
Adds optional new string parameters so we can customize the language in more places on the shared component.